src/PanelTraits/Create.php

```diff
@@ -112,10 +112,12 @@ public function syncPivot($model, $data, $form = 'create')
 
             if (isset($field['morph']) && $field['morph']) {
                 $values = isset($data[$field['name']]) ? $data[$field['name']] : [];
-                if ($model->{$field['name']}) {
-                    $model->{$field['name']}()->update($values);
-                } else {
-                    $model->{$field['name']}()->create($values);
+
+                foreach ($values as $value) {
+                    $class = $field['model'];
+                    $class = new $class();
+                    $ref = $class::find($value);
```

Comment:
What **is** `$class` here:

a) A string?
b) An instantiated object?

It looks like it's an instantiated object here ... and although PHP (being icky) probably allows static calls on instantiated objects it's a bit ick imho.